script/client/spring/application.properties

```diff
@@ -21,7 +21,9 @@ seata.tx-service-group=my_test_tx_group
 seata.enable-auto-data-source-proxy=true
 seata.data-source-proxy-mode=AT
 seata.use-jdk-proxy=false
-seata.client.rm.async-commit-buffer-limit=1000
+
+
```

Comment:
verify that our configuration synchronization script supports blank lines

Reply:
There are many blank lines in this configuration file before. I think it should be supported.